cmd/mt-whisper-importer-writer/verify.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python
+
+
+# this is a tool to verify if the data imported via the whisper importer is
+# actually correct. it uses two inputs, the output of whisper-dump.py and the
+# output of mt-store-cat.
+#
+# pipe the whisper-dump.py output into a file:
+# $> whisper-dump.py storage/whisper/some/id/of/a/metric/1.wsp  > /tmp/whisper_dump
+#
+# pipe the mt-store-cat output into another file:
+# $> mt-store-cat -print-ts -from 1487764800 -to 1487785801 normal id 1.d588ebb28e4d2ca64d828cb1eb6066d4 5h > /tmp/mt_store_dump
+#
+# compare the two:
+# $> ./verify.py /tmp/whisper_dump /tmp/mt_store_dump
+#
+#
+# averaged aggregations are a special case. the whisper importer utility
+# converts avg archives into a sum and a cnt archive and inserts them as such.
+# the cnt archive only gets the aggregation span at each timestamp, the sum
+# archive gets the value from whisper * aggregation span, out of these two
+# series metrictank calculcates the average on the fly when it's queried.
+# so to verify the data we need to use the multiplicator parameter, let's say
```

Comment:
Ok, I'm updating the comment. divider/multiplicator depends on which number we start with, let's just call it factor :)